### PR TITLE
react-962-survey-results-ui-enchancement

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Survey Results - UI Enhancement
[#962](https://github.com/nerubia/kh360-react/issues/962)
BE: https://github.com/nerubia/kh360-nodejs/pull/561

### CODE REVIEW

File: src/features/admin/survey-results/[id]/tables/respondent-view-table.tsx [hunk 1]
```diff
@@ -19,10 +21,10 @@ export const RespondentViewTable = () => {
         totalAmount += parseInt(answer.survey_template_answers?.amount as string)
       }
 
-      return totalAmount
+      return totalAmount.toLocaleString()
     }
   }
-
+  console.log("survey_results", survey_results)
```
Comment (gborgonia-ir) [inline src/features/admin/survey-results/[id]/tables/respondent-view-table.tsx:27]: can pls remove console

Comment (chadiegilnerubia) [reply src/features/admin/survey-results/[id]/tables/respondent-view-table.tsx:27]: ops xD

**Review by emmaacha — APPROVED** [review]

**Review by emmaacha — CHANGES_REQUESTED** [review]
Hello! I added the needed changes for this ticket (updated backend too), kindly just add or improve some minor details like the amount should be in currency form (comma in between numbers), and the column widths, etc.). Thanks!

File: src/features/admin/survey-results/[id]/tables/respondent-view-table.tsx [hunk 2]
```diff
@@ -33,43 +35,52 @@ export const RespondentViewTable = () => {
...
-                        <tr className='hover:bg-slate-100'>
-                          <td className='py-1 px-2 border-b w-1/4 text-center'>
+                        <tr className='hover:bg-slate-100 text-sm'>
+                          <td className='py-1 border-b w-1/12'>{index + 1}</td>
+                          <td className='py-1 border-b w-2/12'>
                             {result.users?.last_name}, {result.users?.first_name}
                           </td>
-                          <td className='py-1 px-2 border-b text-center w-1/6'>
+                          <td className='py-1 border-b w-1/12'>
+                            <Badge variant='green' size='small'>
```
Comment (gborgonia-ir) [inline src/features/admin/survey-results/[id]/tables/respondent-view-table.tsx:79]: pls modify to show variant based on status. you can check getEvaluationResultStatusVariant

File: src/features/admin/survey-results/[id]/tables/respondent-view-table.tsx [hunk 3]
```diff
@@ -92,6 +103,85 @@ export const RespondentViewTable = () => {
                     ))}
                   </tbody>
                 </table>
+                <div className='mt-5 flex flex-col gap-4'>
+                  <p className='text-primary-500 font-bold text-lg'>Companions</p>
```
Comment (gborgonia-ir) [inline src/features/admin/survey-results/[id]/tables/respondent-view-table.tsx:107]: also can pls hide if companion_survey_results is empty

**Review by emmaacha — APPROVED** [review]

**Review by gborgonia-ir — APPROVED** [review]

**Pull request merged**